src/fhirdefs/load.ts

```diff
@@ -113,6 +113,51 @@ export async function loadDependency(
   return FHIRDefs;
 }
 
+/**
+ * Loads custom resources defined in ig-data into FHIRDefs
+ * @param {string} input - The input path to the cli
+ * @param {FHIRDefinitions} defs - The FHIRDefinitions object to load definitions into
+ */
+export function loadCustomResources(input: string, defs: FHIRDefinitions): void {
+  // Similar code for loading custom resources exists in IGExporter.ts addPredefinedResources()
+  const pathEnds = [
+    'capabilities',
+    'extensions',
+    'models',
+    'operations',
+    'profiles',
+    'resources',
+    'vocabulary',
+    'examples' // Must come last in case examples are of other resources
+  ];
+  for (const pathEnd of pathEnds) {
+    let xmlFile = false;
+    let invalidFile = false;
+    const dirPath = path.join(input, 'ig-data', 'input', pathEnd);
+    if (fs.existsSync(dirPath)) {
+      const files = fs.readdirSync(dirPath);
+      for (const file of files) {
+        let resourceJSON: any;
+        if (file.endsWith('.json')) {
+          resourceJSON = fs.readJSONSync(path.join(dirPath, file));
+        } else {
+          xmlFile = xmlFile || file.endsWith('.xml');
+          invalidFile = true;
+          continue;
+        }
+        defs.add(resourceJSON);
```

Comment:
Perhaps this should be:
```ts
if (pathEnd !== 'examples') {
  defs.add(resourceJSON);
}
```
I'm not sure if examples would cause any harm being in there -- but I'm 99.99% sure they would not cause any harm _not_ being in there!

Reply:
They actually don't get loaded, `defs.add` only will add things of `resourceType` `StructureDefinition`, `ValueSet`, `CodeSystem`, or `ImplementationGuide`. But I guess that is not entirely clear, and it is possible the behavior of `defs.add` will change, so I will add that check just to make things more explicit.